plugin/src/code_path_analysis/code_path_analyzer.rs

```diff
@@ -1098,7 +1098,7 @@ mod tests {
                 }])
                 .build()
                 .unwrap(),
-            tree_sitter_lint::tree_sitter_grep::SupportedLanguage::Javascript,
+            tree_sitter_lint::tree_sitter_grep::SupportedLanguageLanguage::Javascript,
```

Comment:
I updated the `tree_sitter_lint`/`tree_sitter_grep` entry points to take `SupportedLanguageLanguage`'s (vs `SupportedLanguage`'s) where that seemed easiest/appropriate